melissa/actions/weather.py

```diff
@@ -1,4 +1,4 @@
-import pywapi
+from melissa.utilities import pywapi
```

Comment:
I must say that pywapi module test isn't required as here we are using the pywapi clone to add the functionality and because of this reason, the test for `melissa/actions/weather.py` at `melissa/tests/test_actions/test_weather.py`will give the result that if the `melissa/tests/test_weather.py` test PASSES then its obvious pywapi module is working.